File: dwaplatform/src/main/java/com/dwaplatform/android/iban/db/Iban.java
package com.dwaplatform.android.iban.db;

import com.dwaplatform.android.db.PlatformDB;
import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;

/**
 * Created by ingrid on 04/01/18.
 */

@Table(database = PlatformDB.class)
public class Iban extends BaseModel {
    @PrimaryKey
    public String id;
    @Column
    public String number;
    @Column
    public String state;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getNumber() {
        return number;
    }

    public void setNumber(String number) {
        this.number = number;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }
}

File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/iban/db/Iban.kt
package com.fintechplatform.ui.iban.db

import com.fintechplatform.ui.db.PlatformDB
import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel

@Table(database = PlatformDB::class)
data class Iban(@PrimaryKey var id: String? = "",
                @Column var number: String? = "",
                @Column var state: String? = "",
                @Column var accountId: String = "") : BaseModel()